src/diazo/utils.py

```diff
@@ -82,7 +82,8 @@ def __call__(self, *args, **kw):
 
 
 def pkg_parse(name, parser=None):
-    with open(pkg_resources.resource_filename("diazo", name)) as f:
+    path = str(files("diazo") / name)
+    with open(path) as f:
         return etree.parse(f, parser=parser)
```

Comment:
```suggestion
    path = files("diazo") / name
    with path.open() as f:
        return etree.parse(f, parser=parser)
```

Reply:
yeah, I was thinking of that, but at the same time importlib.resources mentions that it could potentially be a temporal file extracted out of a zip file, etc, so I was wary to mess too much with it.

See https://importlib-resources.readthedocs.io/en/latest/migration.html

Reply:
path is path, does not matter if its temp extracted. Effectivly we do not have zip-safe anyway, I think zip-safe is dead anyway.